bridges-2/README.md

```diff
@@ -0,0 +1,63 @@
+# Installing Dedalus
+
+To install Dedalus, load the following modules:
+
+gcc/10.2.0
+openmpi/4.0.5-gcc10.2.0
+fftw/3.3.8
+anaconda3/2022.10
+
+and then run the install script provided in this repo (dedalus_hpc_install/bridges-2/conda_install_dedalus2.sh)
+
+It is probably useful to add the following lines to your .bashrc to make loading and unloading modules easy:
+
+    module load gcc/10.2.0
+    module load openmpi/4.0.5-gcc10.2.0
+    module load fftw/3.3.8
+    module load anaconda3/2022.10
+    
+    dedalus2 () {
+	    module load anaconda3/2022.10
```

Comment:
There are some mismatches between tabs and spaces in these functions; looks weird when it formats on github -- try resolving?